blocks/index.ts

```diff
@@ -1,6 +1,9 @@
-import { CallToAction, Component as cta } from './CallToAction';
-import { Content, Component as content } from './Content';
-import { Image, Component as image } from './Image';
+import { CallToAction } from './CallToAction';
+import { Component as cta } from './CallToAction/Component';
```

Comment:
We should split these import/exports into 2 files, one for the block configs and one for the block components. Exporting them all in the same file can cause issues when building in some cases.

We should use more specific named exports and imports to avoid confusion. For example you could add a suffix to the exports, i.e. `CallToActionConfig` and `CallToActionComponent` or similar!